### PR TITLE
Verifone status batch

### PR DESCRIPTION
**Pull request by ncjones**

### CODE REVIEW

File: src/_plugins/optional.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      super
+    end
+
+    def render(context)
+      %{<span class="cp-badge">Optional</span>}
+    end
+  end
+end
+
+Liquid::Template.register_tag('opt', Jekyll::OptionalTag)
```
Comment (heymatthew) [inline src/_plugins/optional.rb:14]: How is this tag used? I can't see examples of it in the code yet :)

**Review by maticko — APPROVED** [review]

File: src/api/batch/batches.md [hunk 2]
```diff
@@ -64,11 +64,11 @@ The following table describes the Batch Types supported for loading.
     </tr>
   </thead>
   <tbody>
-    {%- assign batch_types = site.html_pages | where_exp:"item", "item.permalink contains '/api/batch-types/'" -%}
+    {%- assign batch_types = site.html_pages | where_exp:"item", "item.batch_type != nil" -%}
     {%- for batch_type in batch_types -%}
     <tr>
       <td>
-        <a href="{{ batch_type.url | absolute_url }}">{{ batch_type.type }}</a>
+        <a href="{{ batch_type.url | absolute_url }}">{{ batch_type.batch_type }}</a>
```
Comment (heymatthew) [inline src/api/batch/batches.md:71]: So we're iterating batch_types to get batch_type.batch_type, maybe we could use other language here?

```
     {%- assign batch_pages = ... -%}
     {%- for page in batch_pages -%}
         <a href="{{ page.url | absolute_url }}">{{ page.batch_type }}</a>
```

**Comment by heymatthew** [comment]
Heads up @ncjones , this was merged so you might've missed these comments.

File: src/api/batch/batches.md (hunk 2 above)
Comment (ncjones) [reply src/api/batch/batches.md:71]: @heymatthew yeah it's a batch type page. Could rename to `page.batch_type`.

**Pull request merged**